src/renderers/art/ReactARTFiberEntry.js

```diff
@@ -509,9 +509,7 @@ const ARTRenderer = ReactFiberReconciler({
     return emptyObject;
   },
 
-  scheduleAnimationCallback: ReactDOMFrameScheduling.rAF,
-
-  scheduleDeferredCallback: ReactDOMFrameScheduling.rIC,
+  scheduleCallback: ReactDOMFrameScheduling.rIC,
```

Comment:
Let's just call it `scheduleDeferredCallback` for now but idk how descriptive that actually is.